script/test

```diff
@@ -12,6 +12,6 @@ for pkg in $LINT_PKGS; do
     golint $pkg | grep -v comment
 done
 
-go vet $(glide nv)
+go vet ./...
 goimports -d $(find . -type f -name '*.go' -not -path "./vendor/*")
-go test -race -cover $(glide nv) $@
+go test -race -cover ./... $@
```

Comment:
Any reason to not follow the pattern here? https://github.com/lyft/go-lyft-configset/pull/107/files#diff-b67911656ef5d18c4ae36cb6741b7965L3